ament_index_python/ament_index_python/resources.py

```diff
@@ -22,16 +23,12 @@
 class InvalidResourceTypeNameError(ValueError):
     """Raised when a resource type name is invalid."""
 
-    pass
-
```

Comment:
I'll suggest not removing this in this PR either; it is not directly related to typing.